cli.go

```diff
@@ -19,8 +19,15 @@ func main() {
 	var installOpenAsar = flag.Bool("install-openasar", false, "Install OpenAsar on a Discord install")
 	var uninstallOpenAsar = flag.Bool("uninstall-openasar", false, "Uninstall OpenAsar from a Discord install")
 	var updateFlag = flag.Bool("update", false, "Update your local Vencord files")
+	var dir = flag.string("dir", false, " Select the location of your Discord client")
+	var client = flag.string("client", "default", "Select the branch of Discord you wish to modify [default|stable|ptb|canary]")
 	flag.Parse()
-
+	if *dir != "" && **client != "" {
+		return "", errors.New("the dir and client switches are mutally exclusive")
+	}
+	if *client && ! *client == "default" || *client == "stable" || *client == "ptb" || *client == "canary" ||) {
```

Comment:
**suggestion:** consider `*client != "default"`, I think it's a bit clearer than `! *client == "default"`.

It also looks to me like you're missing an open parentheses and you've got an extra `||` operator at the end of this line. I don't think this will compile as-is.